src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -113,8 +120,15 @@ public URLClassLoader(URL[] urls) {
      * factory argument will be used as the stream handler factory to
      * obtain protocol handlers when creating new jar URLs.
      *
+     * @apiNote If the {@code parent} is specified as {@code null} (for the
+     * bootstrap class loader) then there is no guarantee that all platform
+     * classes are visible.
+     * See {@linkplain ClassLoader##builtinLoaders Run-time Built-in Class Loaders}
+     * for information on the system class loader and other built-in class loaders.
```

Comment:
Seems more relevant to say "for information on the bootstrap class loader and other built-in class loaders." since you specifically mention bootstrap loader in the first sentence.